index.d.ts

```diff
@@ -0,0 +1,112 @@
+declare module 'react-native-touch-id' {
+    /**
+     * The supported biometry type
+     */
+    type BiometryType = 'FaceID' | 'TouchID';
+  
+    /**
+     * Base config to pass to `TouchID.isSupported` and `TouchID.authenticate`
+     */
+    interface Config {
```

Comment:
Can you rename this to be IsSupportedConfig - just to be a bit more clear

Reply:
Good idea!